storage/cluster.yml

```diff
@@ -0,0 +1,22 @@
+apiVersion: rook.io/v1alpha1
+kind: Cluster
+metadata:
+  name: rook
+  namespace: rook
+spec:
+  versionTag: master
+  dataDirHostPath: /storage
+  placement:
+    all:
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Equal
+        effect: NoSchedule
+  storage:
+    useAllNodes: true
+    useAllDevices: false
+    # deviceFilter: vdb
+    storeConfig:
+      storeType: filestore
+      databaseSizeMB: 1024
```

Comment:
what type of environments will this be deployed to?  if it will always have normal sized disks (100 GB+) then you can remove these `dataSizeMB` and `journalSizeMB` fields.  We only use them in our development VM environments where the disks are ~5GB (tiny!)

Reply:
Within the context of this project, this seems to be a good configuration. The idea is to provide hyper-converged storage, typically in the range of 10-50GB. What size would you recommend in this case?